src/@sdk/mutations/index.ts

```diff
@@ -6,6 +6,11 @@ import gql from "graphql-tag";
 
 import * as Address from "./address";
 import * as Auth from "./auth";
+import * as Checkout from "./checkout";
+import {
+  createCheckout,
+  createCheckoutVariables
```

Comment:
TokenAuth, TokenAuthVariables are used with capital first letter. I also used capital letter for my mutations - imho we should keep consistency as this will be our SDK :) I dont mean that we have to use capital letters but we should be consisent :)

Reply:
Good point 👍